examples/petsc/multigrid.c

```diff
@@ -578,7 +641,7 @@ int main(int argc, char **argv) {
 
     ierr = PetscViewerCreate(comm, &vtkviewersoln); CHKERRQ(ierr);
     ierr = PetscViewerSetType(vtkviewersoln, PETSCVIEWERVTK); CHKERRQ(ierr);
-    ierr = PetscViewerFileSetName(vtkviewersoln, "solution.vtk"); CHKERRQ(ierr);
+    ierr = PetscViewerFileSetName(vtkviewersoln, "solution.vtu"); CHKERRQ(ierr);
```

Comment:
Should we go ahead and do this in every PETSc example?

Reply:
Yeah, we never should have written `.vtk` (a slow and limited format what was deprecated almost 20 years ago).